src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -351,3 +355,36 @@ def body(self):
             JMESPathCheck('[0].name', self.webapp + '/' + slot),
             ])
         self.cmd('appservice web deployment slot delete -g {} -n {} --slot {}'.format(self.resource_group, self.webapp, slot), checks=NoneCheck())
+
+class WebappSSLCertTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(WebappSSLCertTest, self).__init__(__file__, test_method, resource_group='azurecli-webapp-ssl')
```

Comment:
Recommend the resource_group name `test_cli_webapp_ssl`.

It makes it easier for us to identify test resource groups that didn't get cleaned up for whatever reason to simply search with the 'test_cli_' prefix.